lib/onebox/engine/google_photos_onebox.rb

```diff
@@ -9,51 +9,50 @@ class GooglePhotosOnebox
 
       def to_html
         og = get_opengraph
-        return video_html(og) if !Onebox::Helpers::blank?(og[:video_secure_url])
-        return album_html(og) if !Onebox::Helpers::blank?(og[:type]) && og[:type] == "google_photos:photo_album"
-        return image_html(og) if !Onebox::Helpers::blank?(og[:image])
+        return video_html(og) if !og.video_secure_url.nil?
+        return album_html(og) if !og.type.nil? && og.type == "google_photos:photo_album"
+        return image_html(og) if !og.image.nil?
         nil
       end
 
       private
 
       def video_html(og)
-        escaped_src = ::Onebox::Helpers.normalize_url_for_output(og[:video_secure_url])
-        escaped_image_src = ::Onebox::Helpers.normalize_url_for_output(get_secure_link(og[:image]))
+        escaped_image_src = get_secure_link(og.image)
```

Comment:
Would it make sense to add a `og.secure_image` method? I think the `get_secure_link` code is duplicated multiple times in the code base?